climateconnect/climateconnect
# Add restricted_profile boolean field to restrict user access

## What and Why

Partially solved #1144. 

I have added a `restricted_profile` boolean flag to restrict access of users who are abusing the platform. I would like to note that it does not solve the problem completely but gives us a breathing room while we work on a permanent  solution.

# Test plan
* Use django shell or django admin, set `user_profile.restricted_profile` to `True`. 
* Go to http://localhost:3000/inbox and start chat with another user. 
* Write a message and try to send it. 
* You should see a message at the top that you account has been restricted. Here is a screenshot.
![Screenshot 2022-12-15 at 8 21 17 AM](https://user-images.githubusercontent.com/15343711/207913669-b266bb22-1935-4061-bbdd-0a51234871e8.png)
